worker/caasadmission/admission_test.go

```diff
@@ -4,12 +4,11 @@
 package caasadmission_test
```

Comment:
@tlm The tests in this package are being run twice.

I think you need to add a _package_test.go_ here.

Please ensure that all _package_test.go_ files are in {package} rather than {package_test}